spec/automation/unit/method_validation/launch_ansible_job_spec.rb

```diff
@@ -94,4 +118,17 @@
     LaunchAnsibleJob.new(service).main
     expect(service.get_state_var(:ansible_job_id)).to eq(job.id)
   end
+
+  it "get dialog parameters" do
+    prov_options[:dialog_param_name] = 'fred'
+    root_object[:job_template_name] = job_template.name
+    root_object[:miq_provision] = svc_provision
+    ext_vars['name'] = 'fred'
+    current_object = MiqAeMockObject.new('param1' => 'x=X', 'param2' => 'y=Y')
```

Comment:
You're forced to create `'param1' => 'x=X', 'param2' => 'y=Y')` because of the way you initialize `ext_vars` even though your testing purpose here is not for `param1` and `param2`. Think of clearing `extra_vars` so that you don't have to setup `param1` and `param2` every time.